modules/hcp-aws-deploy-hc.adoc

```diff
@@ -43,6 +44,9 @@ $ hcp create cluster aws \
 <7> Specify the node pool replica count, for example, `3`.
 <8> By default, all `HostedCluster` and `NodePool` custom resources are created in the `clusters` namespace. You can use the `--namespace <namespace>` parameter, to create the `HostedCluster` and `NodePool` custom resources in a specific namespace.
 <9> Specify the Amazon Resource Name (ARN), for example, `arn:aws:iam::820196288204:role/myrole`.
+<10> If you want to indicate whether the EC2 instance runs on shared or single tenant hardware, include this field. The `--render-into` flag renders Kubernetes resources into the YAML file that you specify in this field. Then, continue to the next step to edit the YAML file.
+
+. In the command to create the hosted cluster, if you included the `--render-into` flag and specified a YAML file, edit the `NodePool` specification in the YAML file to indicate whether the EC2 instance should run on shared or single-tenant hardware.
```

Comment:
maybe include a snippet of the yaml where the user should add the new field
```yaml
apiVersion: hypershift.openshift.io/v1beta1
kind: NodePool
metadata:
  name: <nodepool-name>
spec:
  platform:
    aws:
      placement:
        tenancy: one of "default", "dedicated" or "host"
```

Reply:
Awesome. I was hoping you would provide an example like that 😄